pkg/operator/status.go

```diff
@@ -72,6 +72,10 @@ func (optr *Operator) syncRelatedObjects() error {
 		{Group: "machineconfiguration.openshift.io", Resource: "machineconfigs"},
 		// gathered because the machineconfigs created container bootstrap credentials and node configuration that gets reflected via the API and is needed for debugging
 		{Group: "", Resource: "nodes"},
+		{Resource: "namespaces", Name: "openshift-kni-infra"},
```

Comment:
could you add a comment above the new additions just to memorialize what this is for?

Reply:
Done